Magento_Theme/templates/header/marketing-bar.phtml

```diff
@@ -1,11 +1,16 @@
 <div class="marketing-bar">
-    Subscribe and get 10$ for Shopping!
-    <button class="button button--icon marketing-bar__close"
-            type="button"
-            aria-label="<?= /* @escapeNotVerified */ __('Close marketing bar') ?>"
-    >
-        <svg title="close" class="marketing-bar__close-icon">
-            <use xlink:href="<?= $block->getViewFileUrl('/images/icons-sprite.svg#close'); ?>"></use>
-        </svg>
-    </button>
+    <div class="container marketing-bar__container">
+        <div class="marketing-bar__text">
+            <!-- TODO: Move to CMS block -->
+            Subscribe and get 10$ for Shopping!
```

Comment:
Shouldn't this go from translate function ?

Reply:
That's why it's marked as "move to CMS block", b/c it's just placehoder